app/frontend/src/features/profile/hooks/useUpdateUserProfile.ts

```diff
@@ -34,7 +32,7 @@ export default function useUpdateUserProfile() {
         setMutationError(null);
       },
       onSuccess: () => {
-        queryClient.invalidateQueries(["user", userId]);
```

Comment:
This invalidation needs to also be kept in addition to the `accountInfoQueryKey`, otherwise user data is gonna be outdated after they updated their profile and introduces an unintended bug...

Reply:
My fault, I thought this new line overrides the previous line. I should have verified that before committing this merge.